src/iadmin.cpp

```diff
@@ -2574,23 +2752,57 @@ usage( char *subOpt ) {
                        "help", "h"
                       };
 
-    char **pMsgs[] = { luMsgs, luaMsgs, luanMsgs, luzMsgs, ltMsgs, lrMsgs,
-                       lsMsgs, lzMsgs,
-                       lgMsgs, lgdMsgs, mkuserMsgs,
-                       moduserMsgs, auaMsgs, ruaMsgs, rppMsgs,
-                       rmuserMsgs, mkdirMsgs, rmdirMsgs, mkrescMsgs,
-                       modrescMsgs, modrescDataPathsMsgs, rmrescMsgs,
-                       addchildtorescMsgs, rmchildfromrescMsgs,
-                       mkzoneMsgs, modzoneMsgs, modzonecollaclMsgs, rmzoneMsgs,
-                       mkgroupMsgs, rmgroupMsgs, atgMsgs,
-                       rfgMsgs, atMsgs, rtMsgs, spassMsgs, dspassMsgs,
-                       ctimeMsgs,
-                       suqMsgs, sgqMsgs, lqMsgs, cuMsgs,
-                       rumMsgs, asqMsgs, rsqMsgs, modrepl_usage,
-                       get_delay_server_info_usage,
-                       set_delay_server_usage,
-                       helpMsgs, helpMsgs
-                     };
+    char** pMsgs[] = {luMsgs,
+                      luaMsgs,
+                      luanMsgs,
+                      luzMsgs,
+                      ltMsgs,
+                      lrMsgs,
+                      lsMsgs,
+                      lzMsgs,
+                      lgMsgs,
+                      lgdMsgs,
+                      mkuserMsgs,
+                      moduserMsgs,
+                      auaMsgs,
+                      ruaMsgs,
+                      rppMsgs,
+                      rmuserMsgs,
+                      mkdirMsgs,
+                      rmdirMsgs,
+                      mkrescMsgs,
+                      modrescMsgs,
+                      modrescDataPathsMsgs,
+                      rmrescMsgs,
+                      addchildtorescMsgs,
+                      rmchildfromrescMsgs,
+                      mkzoneMsgs,
+                      modzoneMsgs,
+                      modzonecollaclMsgs,
+                      rmzoneMsgs,
+                      mkgroupMsgs,
+                      rmgroupMsgs,
+                      atgMsgs,
+                      rfgMsgs,
+                      atMsgs,
+                      rtMsgs,
+                      spassMsgs,
+                      dspassMsgs,
+                      ctimeMsgs,
+                      suqMsgs,
+                      sgqMsgs,
+                      lqMsgs,
+                      cuMsgs,
+                      rumMsgs,
+                      asqMsgs,
+                      rsqMsgs,
+                      modrepl_usage,
+                      get_delay_server_info_usage,
+                      set_delay_server_usage,
+                      get_pam_password_config_usage,
+                      set_pam_password_config_usage,
+                      helpMsgs,
+                      helpMsgs};
```

Comment:
Not sure why this is here twice. Will look into it.

I also think I forgot to add my help messages in all the right places.

Reply:
"help" and "h" are recognized by `iadmin`, which is why there are two of these.

Reply:
well, feels weird, but explained!